core/src/main/resources/schema/infinispan-config-15.1.xsd

```diff
@@ -2400,7 +2400,7 @@
               </xs:documentation>
             </xs:annotation>
           </xs:attribute>
-          <xs:attribute name="wait-time" type="xs:long" default="${XSiteStateTransfer.wait-time}">
+          <xs:attribute name="wait-time" type="xs:string" default="${XSiteStateTransfer.wait-time}">
             <xs:annotation>
               <xs:documentation>
                 Sets the amount of time, in milliseconds, to wait between retry attempts for push state failures. You must specify a value of 1 or more. The default value is 2000.
```

Comment:
You should copy/paste the blurb we already have elsewhere about the supported time quantities.